src/pages/ProductList/ListMenu/ListMenuSort.js

```diff
@@ -0,0 +1,6 @@
+export const menuList = [
```

Comment:
이런 변하지 않는 상수 데이터는 대문자로 작성하는 것이 컨벤션입니다 :) 
아래와 같이 작성해주시면 되겠네요!
```suggestion
export const MENU_LIST = [
```

Reply:
넵!